CHANGES.rst

```diff
@@ -22,13 +22,33 @@ formatted links that link to the relevant place in the code overview.
 
 .. begin_changelog_body
 
-UNRELEASED
+Unreleased
 ----------
  * **[BUGFIX]** Fixed login password prompt issue on windows (#485).
  * **[BUGFIX]** Fixed unicode user-agent issue (#483).
  * **[BUGFIX]** Fix duplicate request issue with comment and submission streams
    (#501).
+ * **[BUGFIX]** Stopped :meth:`praw.objects.Redditor.friend` from raising
+   LoginRequired when using OAuth.
+ * **[BUGFIX]** Stopped a json-parsing error from occuring in cases where
+   reddit's response to a request was an empty string. :meth:`request_json`
+   will now simply return that empty string.
  * **[CHANGE]** Added messages to all PRAW exceptions (#491).
+ * **[CHANGE]** Made it easier to send JSON dumps instead of form-encoded data
+   for http requests. Some reddit api-v1 paths are starting to require this.
```

Comment:
*Nitpick* `this` should be followed by some noun to precisely state what `this` refers to. In this case, 'behavior' may be appropriate.